include/crud-posts.php

```diff
@@ -283,7 +283,10 @@ public function create_media_translation( $post_id, $lang ) {
 		// Create a new attachment ( translate attachment parent if exists ).
 		add_filter( 'pll_enable_duplicate_media', '__return_false', 99 ); // Avoid a conflict with automatic duplicate at upload.
 		unset( $post['ID'] ); // Will force the creation.
-		$post['post_parent'] = ( $post['post_parent'] && $tr_parent = $this->model->post->get_translation( $post['post_parent'], $lang->slug ) ) ? $tr_parent : 0;
+		$tr_parent = $this->model->post->get_translation( $post['post_parent'], $lang->slug );
+		if ( ! $post['post_parent'] || ! $tr_parent ) {
+			$post['post_parent'] = 0;
+		}
```

Comment:
You don't actually assign `$tr_parent` to `$post['post_parent']`.
```suggestion
		if ( ! empty( $post['post_parent'] ) ) {
			$post['post_parent'] = (int) $this->model->post->get_translation( $post['post_parent'], $lang->slug );
		}
```